### ISSUE TITLE
Prioritized experience replay support in the new DQN

### ISSUE THREAD

**Issue by ADGEfficiency** [ADGEfficiency/energy-py#24]
Need to add additional Tensorflow placeholders & graph to integrate with the prioritized experience replay memory.

Mainly around sending the importance weights into the loss function, and functionality to decay the priority replay hyperparams (should this decay be done in tf or np?)
